lua/apisix/core/schema.lua

```diff
@@ -1,32 +1,24 @@
-local json = require('rapidjson')
-local schema_validator = json.SchemaValidator
-local schema_doc = json.SchemaDocument
-local json_doc = json.Document
+local jsonschema = require('jsonschema.init')
+local lrucache = require("apisix.core.lrucache")
 
+local cached_validator = lrucache.new({count = 1000, ttl = 0})
+local opts = {match_pattern = ngx.re.find}
 
-local cached_sd = require("apisix.core.lrucache").new({count = 1000, ttl = 0})
-
-
-local _M = {version = 0.2}
+local _M = {version = 0.3}
 
 
 local function create_validator(schema)
-    local sd = schema_doc(schema)
-    local validator = schema_validator(sd)
-
-    -- need to cache `validator` and `sd` object at same time
-    return {validator, sd}
+    -- local code = jsonschema.generate_validator_code(schema, opts)
+    -- local file2=io.output("/tmp/2.txt")
+    -- file2:write(code)
+    -- file2:close()
```

Comment:
remove debug codes.

Reply:
we can remove it after a while(maybe 1 month)